app/src/main/java/com/codesoom/assignment/repositories/TaskRepository.java

```diff
@@ -0,0 +1,78 @@
+package com.codesoom.assignment.repositories;
+
+import com.codesoom.assignment.errors.TaskIdNotFoundException;
+import com.codesoom.assignment.models.Task;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import org.springframework.stereotype.Repository;
+
+/**
+ * 등록된 할 일들을 저장 및 관리합니다.
+ */
+@Repository
+public class TaskRepository {
+
+    private final Map<Long, Task> taskMap = new HashMap<>();
+
+    /**
+     * 번호와 할 일을 받아 저장합니다.
+     *
+     * @param id 저장할 번호
+     * @param task 저장할 할 일
+     */
+    public void save(Long id, Task task) {
+        taskMap.put(id, task);
+    }
+
+    /**
+     * 모든 할 일을 찾습니다.
+     *
+     * @return 할 일 목록
+     */
```

Comment:
"찾는 것"보다 중요한 것은 이 메소드가 무엇을 리턴하는지에 대한 것입니다.

```suggestion
    /**
     * 모든 할 일을 리턴합니다.
     *
     * @return 할 일 목록
     */
```